tests/sdk_test/sdk_test.cpp

```diff
@@ -3435,6 +3457,18 @@ void MegaChatApiTest::makeContact(unsigned int a1, unsigned int a2)
     mContactRequest[a2] = NULL;
 }
 
+bool MegaChatApiTest::isChatroomUpdated(unsigned int index, MegaChatHandle h)
+{
+    for (size_t i = 0; i < mChatListUpdated[index].size(); i++)
```

Comment:
`mChatListUpdated` is a `std::vector`. You should use `for (auto &chatid : mChatListUpdate)`